client/src/components/profileInfoEdit/ProfileResetModal.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * Delete Alert Modal component display a modal and ask for confirmation when the delete button is clicked
+ */
+
+import React from 'react';
+import { Button, Modal } from 'react-bootstrap';
+import { useSelector, useDispatch } from 'react-redux';
+import { PropTypes } from 'prop-types';
+import { logout } from '../../actions/auth';
+import { resetTeamData, deleteGHPages } from '../../actions/team';
+import { resetHomepage } from '../../actions/homepage';
+import { resetWebPage } from '../../actions/website';
+
+const ProfileResetModal = ({ resetAlert, setResetAlert, type }) => {
+  const { teamId } = useSelector((state) => state.team);
+  const dispatch = useDispatch();
+
+  const isDeleteFlag = type;
+  const titleMessage = isDeleteFlag ? 'Delete Account Data!' : 'Reset Account Data!';
+  const warningMessage = isDeleteFlag ? 'Are you sure you want to delete your account? ' : 'Are you sure you want to reset your account? ';
+  const buttonName = isDeleteFlag ? 'Delete' : 'Reset';
+
+  const HandleResetType = () => {
+    const accessToken = localStorage.getItem('GH_access_token');
+    dispatch(resetHomepage(teamId));
+    dispatch(resetWebPage(teamId));
+    dispatch(deleteGHPages(teamId, accessToken));
+
+    // changes based on button clicked at run time
+    dispatch(resetTeamData(teamId, isDeleteFlag));
+
+    if (isDeleteFlag) {
+      dispatch(logout());
+    } else {
+      setResetAlert(false);
+    }
+  };
+
+  return (
+    <Modal show={resetAlert}>
+      <Modal.Header className="modalHeader">
+        <Modal.Title>
+          {titleMessage}
+        </Modal.Title>
+      </Modal.Header>
+      <Modal.Body>
+        {warningMessage}
+        All significant data will be deleted!
```

Comment:
`All the data associated with your account will be deleted along with your account itself.`?